includes/Context.php

```diff
@@ -510,4 +510,26 @@ public function get_locale( $context = 'site', $format = 'default' ) {
 
 		return $wp_locale;
 	}
+
+	/**
+	 * Check whether the site has auto updates enabled for this plugin.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool `true` if auto updates are enabled, otherwise `false`.
+	 */
+	public function is_autoupdate_enabled() {
```

Comment:
I'm not sure this makes sense to be in `Context`, as this is only used in one place. I think we only need it for the JS data we pass off to the front end, correct?

Reply:
For now, yes. What would be a better place?

Reply:
A helper method on the class where we populate that data would be fine to start with, although that would be adding to `Authentication` in this case which is already quite large. What do you think about extracting this in a new `Core\Util\Auto_Updates` class? We probably won't need this method elsewhere and we should be able to filter the data into the PHP -> JS data.

Reply:
Makes, sense. Will add it to `Core\Util\Auto_Updates`.